Automattic/wp-calypso
# Automattic for Agencies: Implement fetch agencies & set current agency.

Resolves https://github.com/Automattic/jetpack-genesis/issues/281

## Proposed Changes

This PR implements 

- Fetch agencies when A4A is loaded every time.
- Set the current agency when agencies are fetched, which will allow us to support multiple agencies in the future.

## Testing Instructions

- Switch to the branch `git checkout add/a4a/implement-fetch-agencies`.
- Open the Calyso live link for A4A.
- Follow the steps here to create the agency ID: D139529-code.
- Verify that an API call is being made to fetch all the agencies: `/agency,`, and it returns all the agencies. 
- Click the Marketplace menu item > Select Jetpack Complete and issue it > Verify that the selected licenses are issued > Assign the issued licenses to a site by selecting one from the Assign License page > Verify that the licenses are assigned to the selected site.

## Pre-merge Checklist
